docs/tasks.md

```diff
@@ -153,3 +139,179 @@ work has completed, spinning up replacement workers.
+
+
+#### Handle Setting Changes
+
+Any time you change a setting in Tower (_e.g._, in `api/v2/settings`), data will be added to or altered in a database. Since querying databases directly can be extremely time-consuming, each node in a cluster runs a local `memcached` server, none of which are aware of each other. They all potentially have different values contained within, but ultimately need to be consistent. So how can this be accomplished?
+
+"Handle Settings Changes" provides the solution!  This "fanout" task (_i.e._, all nodes execute it) makes it so that there is a single source of truth even within a clustered system.  When anything gets altered or updated in the database, all of the `memcached` servers on each node needs to "forget" the value that they previously retained; with this task, whenever `perform_update()` or `perform_destroy` gets invoked in `awx/conf/views.py`, this task will clear any associated values within each node's `memcached` process and ensure that all of the nodes in the cluster have the most up-to-date information in their caches at all times.
+
+
+### Analytics and Administrative Tasks
+
+#### Profile SQL
+
+This task was added as a new feature in Tower 3.5. It allows the user to turn on a global profiler in their system, so that Tower can profile all of the SQL queries that they make.  This is a "fanout" style task (meaning all nodes execute it), and one of the main benefits is that it assists with identifying slow queries.
+
+
+#### Gather Analytics
+
+[This is a Tower-only feature and requires a subscription to RH Insights.]
+
+The analytics collection `gather()` and `ship()` functions are called by an `awx-manage gather_analytics --ship` command, which runs on whichever Tower instance it is invoked on. When these functions are called by Celery beat (currently at midnight local time), it is run on one `execution_node`, or Tower instance, by the Python in the AWX virtualenv.
+
+For more details about analytics, please visit the [Usability Analytics and Data Collection](https://docs.ansible.com/ansible-tower/latest/html/administration/usability_data_collection.html) page.
+
+
+#### Run Administrative Checks
+
+Not applicable to AWX, this task checks that the Tower license currently in use is valid and alerts the admin user(s) via email when they are in danger of going over capacity and/or when the license is about to expire. Specifically (in cases of going over capacity), it triggers when the node count is at or over 90% of what the license allows.
+
+
+#### Purge Old Stdout Files
+
+Previously, AWX/Tower wrote `stdout` to a disk.  With the implementation of this task, old files are now cleaned on a regular basis when the `LOCAL_STDOUT_EXPIRE_TIME` is triggered.
```

Comment:
I gave you some incorrect info on this one.  It's still _technically_ in use.  Let's say something like:

```diff
- Previously, AWX/Tower wrote `stdout` to a disk.
+ AWX periodically sometimes buffers stdout for playbook runs to disk _when users download stdout for a job run.  This task implements a periodic cleanup of the directory where this data is stored.
```